### DISCUSSION TITLE
What about integrating renovate for Robolectric's dependency management

### DISCUSSION THREAD

**Discussion by utzcoz** [robolectric/robolectric#7187] [Ideas]
Looks like [renovate](https://github.com/apps/renovate) is more powerful than dependabot, and it also can help to upgrade version of GitHub Actions. IMHO, it can help us to get rid of frequently and important dependency management work.

**Comment by Goooler** [comment]
Might migrate to version catalogs first.

**Reply by utzcoz** [reply]
Thanks for pointing out it. @Goooler do you have interested in enabling version catalogs for Robolectric? I have noticed you have did it great for other awesome open-source projects. If you would like to do it for Robolectric too, I can create an issue and assign it to you. Thanks your work for Robolectric.

**Reply by Goooler** [reply]
Ok, have a try.

**Reply by utzcoz** [reply]
Thanks. I created an issue for it: https://github.com/robolectric/robolectric/issues/7196. Feel free to send PR for it.
